packages/kit/src/vite/index.js

```diff
@@ -318,14 +318,43 @@ function kit() {
 				})};\n`
 			);
 
+			// TODO where is this used?
 			process.env.SVELTEKIT_SERVER_BUILD_COMPLETED = 'true';
```

Comment:
unrelated to this PR, but pretty sure we can get rid of this?

Reply:
No objections assuming it's not used anywhere. It was an internal-only thing, so we shouldn't worry about anyone else relying on it